packages/shared/lib/core/network/interfaces/connected-chain.interface.ts

```diff
@@ -0,0 +1,7 @@
+import { NetworkHealth } from '../enums'
+
+export interface ConnectedChain {
```

Comment:
Should be `IConnectedChain`, but this interface might go away anyway one we add the classes in b/c we'll have the metadata available from that then also query the status in real time from the class. We can keep this for now (still rename it though pls :pray:)